libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -181,12 +181,14 @@ private static void processMPEG4FmtpAttribute(
       Format.Builder formatBuilder, ImmutableMap<String, String> fmtpAttributes) {
     @Nullable String configInput = fmtpAttributes.get(PARAMETER_MP4V_CONFIG);
     if (configInput != null) {
-      byte[] csd = Util.getBytesFromHexString(configInput);
-      formatBuilder.setInitializationData(ImmutableList.of(csd));
+      byte[] configBuffer = Util.getBytesFromHexString(configInput);
+      formatBuilder.setInitializationData(ImmutableList.of(configBuffer));
       Pair<Integer, Integer> resolution =
-          CodecSpecificDataUtil.getVideoResolutionFromMpeg4VideoConfig(csd);
-      formatBuilder.setWidth(resolution.first);
-      formatBuilder.setHeight(resolution.second);
+          CodecSpecificDataUtil.getVideoResolutionFromMpeg4VideoConfig(configBuffer);
+      formatBuilder.setWidth(resolution.first).setHeight(resolution.second);
+    } else {
+      // set the default width and height
```

Comment:
Why is the default width and height 352x288? Is it in an RFC? If so please add a link

Reply:
For non-standard fmtp where configinput is missing, we use 352X288 as default since CIF is the default resolution used by Android Software decoder. Adding the link for your reference
https://cs.android.com/android/platform/superproject/+/master:frameworks/av/media/codec2/components/mpeg4_h263/C2SoftMpeg4Dec.cpp;l=130